transport/http.go

```diff
@@ -91,11 +91,9 @@ type HTTPTransport struct {
 // streaming data to the APM Server. The returned HTTPTransport will be
 // initialized using the following environment variables:
 //
-// - ELASTIC_APM_SERVER_URLS: a comma-separated list of APM Server URLs.
-//   The transport will use this list of URLs for sending requests,
-//   switching to the next URL in the list upon error. The list will be
-//   shuffled first. If no URLs are specified, then the transport will
-//   use the default URL "http://localhost:8200".
+// - ELASTIC_APM_SERVER_URL: the APM Server URL used for sending
+//   requests. If no URL is specified, then the transport will use the
+//   default URL "http://localhost:8200".
```

Comment:
Just a minor thing, but since `ELASTIC_APM_SERVER_URLS` is deprecated, should we also change the order of how the ENV vars are parsed, so that `ELASTIC_APM_SERVER_URL` is always used if provided.  https://github.com/elastic/apm-agent-go/blob/8b1a0fe81cca0549e5b2013d209f2dcdd34a9aa0/transport/http.go#L528-L538.

Reply:
I did think about doing that, and decided at the time to make it a purely non-functional change. I don't think it's likely to matter either way, but I'd like to avoid a breaking change unless necessary.

Reply:
I assume since you approved that you're good either way, so I'll merge. If you disagree with my decision, let me know and we can come back to it.

Reply:
yes, I am fine either way - thanks for the explanation.